src/main/java/org/testng/TestNG.java

```diff
@@ -533,7 +533,9 @@ public void setTestClasses(Class[] classes) {
   }
 
   public void addMethodSelector(String className, int priority) {
-    m_methodDescriptors.put(className, priority);
+    if (className != null && !className.trim().isEmpty()) {
```

Comment:
Is `trim` really useful?
Maybe a util method `boolean isEmpty(String s)` could be helpful (a method may already exist).

Reply:
Since this is a public method, I chose to include a `trim()` invocation as well. In terms of the utils method, I have refactored to use `org.testng.util.Strings#isNotNullAndNotEmpty`